examples/obs-agent-ocp/version.tf

```
@@ -10,7 +10,7 @@ terraform {
     }
     helm = {
       source  = "hashicorp/helm"
-      version = ">= 2.15.0"
+      version = "<= 3.0.0, <4.0.0"
```

other way around surely? 
```suggestion
      version = ">= 3.0.0, <4.0.0"
```